README.md

```diff
@@ -1,2 +1,181 @@
-# jbang-catalog
+# Jython JBang Catalog
+
 Catalog for the JBang command line launcher (https://jbang.dev)
+
+## Usage
+
+To use the JBang scripts from this catalog you need to have [JBang](https://www.jbang.dev/) installed.
+
+To start the Jython interpreter or run a Jython script via the interpreter, use JBang as follows and add additional options as required:
+
+* `jbang run jython-cli@jython <jython-script>.py`
+
+## App
+
+The `jython-cli@jython` JBang script can also be installed as a JBang application which is accessible from the command-line.
+
+```
+jbang app install jython-cli@jython
+[jbang] Command installed: jython-cli
+```
+
+Jython scripts can be run using the `jython-cli` application as:
+
+* `jython-cli <jython-script>.py`
+
+On supported platforms a Jython script (with execution permissions enabled) can also be started as
+
+`./<jython-script>.py`
+
+provided that the first line of the script contains text:
+
+`#!/usr/bin/env jython-cli`
+
+## PEP-723
+
+`jython-cli` uses the `PEP-723` inline script metadata format to specify:
+
+* The version of Jython to use: ..., 2.7.2, 2.7.3, 2.7.4, ...
```

Comment:
This is an interesting take. I rather expected either

`* The version of Jython to use: 2.7.1, ...`

or

`* The version of Jython to use: 2.7.4, ...`

(depending on how much you would recommend against pre-4 2.7 versions).
Why so complicated?

Reply:
> The version of Jython to use: 2.7.1, ...

Fixed.